docs/01_introduction/hardware.md

```diff
@@ -0,0 +1,112 @@
+# Choosing the Optimal Computer
+
+Though Python is runnable on most of modern operating systems (OS) including Windows, MacOS and Linux, it is beneficial to keep scripting under *nix environment. Here we provide a guide for beginners to choose your computing hardwares.
+
+This guide is intentionally written for programming beginners to code locally. For advance research units equip with python servers, we will cover a series of remote coding techniques to unleash more complex bioimage analysis.
```

Comment:
```suggestion
This guide is intentionally written for programming beginners to code locally.
```

I'm removing the sentence about remote coding, because the mentioned series is not linked yet. We can add the reference back once these tutorials were written.

Reply:
Removed accordingly